amgut/db/ag.dbs

```diff
@@ -22943,20 +22919,35 @@ AS $function$xpath_exists$function$
 				<fk_column name="sample_id" pk="sample_id" />
 			</fk>
 		</table>
+		<table name="sample_plate_study" >
+			<column name="sample_plate_id" type="bigint" jt="-5" mandatory="y" />
+			<column name="study_id" type="bigint" jt="-5" mandatory="y" />
+			<index name="idx_sample_plate_study" unique="NORMAL" >
+				<column name="sample_plate_id" />
+			</index>
+			<index name="idx_sample_plate_study" unique="NORMAL" >
+				<column name="study_id" />
+			</index>
+			<fk name="fk_sample_plate_study" to_schema="pm" to_table="sample_plate" >
+				<fk_column name="sample_plate_id" pk="sample_plate_id" />
+			</fk>
+			<fk name="fk_sample_plate_study_study" to_schema="pm" to_table="study" >
+				<fk_column name="study_id" pk="study_id" />
+			</fk>
+		</table>
 		<table name="study" >
-			<column name="study_id" type="bigserial" jt="-5" mandatory="y" />
-			<column name="qiita_study_id" type="bigint" jt="-5" />
-			<column name="title" type="varchar" jt="12" />
-			<column name="alias" type="varchar" jt="12" />
-			<column name="notes" type="varchar" jt="12" />
+			<column name="study_id" type="bigint" jt="-5" mandatory="y" />
+			<column name="title" type="varchar" jt="12" mandatory="y" />
+			<column name="alias" type="varchar" jt="12" mandatory="y" />
+			<column name="jira_key" type="varchar" jt="12" mandatory="y" />
```

Comment:
Good catch - I've updated the patch but not the dbschema file! Fixed - everything now should say jira_id.